source/automem/ref_counted.d

```diff
@@ -101,7 +103,25 @@ private:
         else
             size_t _count;
 
-        alias _object this;
+        static if (is(Type == class)) {
+            inout(Type) _get() inout {
```

Comment:
Technically, it's not. With DIP1000 it could be, i.e. returning `scope` reference.